### PR TITLE
LPS-99296 SF

### PR DESCRIPTION
**Pull request by brunofarache**

### CODE REVIEW

File: modules/apps/app-builder/app-builder-web/src/main/java/com/liferay/app/builder/web/internal/deploy/StandaloneAppDeployer.java [hunk 1]
```diff
@@ -73,10 +70,16 @@ public void deploy(long appId) throws Exception {
...
-					"App" + appId, appBuilderApp.getCompanyId(),
-					AppBuilderPortletKeys.STANDALONE_APP + "_" + appId)));
+			key -> {
+				try {
+					return _deployAppPortlet(
+						appBuilderApp.getCompanyId(), appId,
+						AppBuilderPortletKeys.STANDALONE_APP + "_" + appId);
+				}
+				catch (PortalException pe) {
+					throw new RuntimeException(pe);
```
Comment (brunofarache) [inline modules/apps/app-builder/app-builder-web/src/main/java/com/liferay/app/builder/web/internal/deploy/StandaloneAppDeployer.java]: @gabrielwas 
Probably better to throw IllegalStateException instead.

**Comment by gabrielwas** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by gabrielwas** [comment]
Pull request submitted to https://github.com/brunofarache/liferay-portal/pull/42. [See changes here.](https://github.com/gabrielwas/liferay-portal/compare/6ab78affc31a~1...6ab78affc31a) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by gabrielwas** [comment]
Pull request submitted to https://github.com/brunofarache/liferay-portal/pull/43. [See changes here.](https://github.com/gabrielwas/liferay-portal/compare/20bc4f4ff270~2...20bc4f4ff270) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**